### ISSUE TITLE
Many notifications when trying to create an Eperson with an already existing email

### ISSUE THREAD

**Issue by corrad82-4s** [DSpace/dspace-angular#1325]
**Describe the bug**
When trying to register an Eperson with an already existing email, the save action does not create a new Eperson, but the form submission is performed and user sees three different error notifications.

![image](https://user-images.githubusercontent.com/70699836/134501391-5328ef58-0784-405b-852e-d76242f33e2c.png)

**To Reproduce**
1. Login as administrator on dspace7 demo
2. go to Access Control -> People
3. click "add Eperson"
4. fill the form with an email of an already existing user

**Expected behavior**
Three different notifications might be a bit confusing for the user, we believe he / she should be  warned about the already existing email, but with a single error message, or even better, made unable to perform the "save" action if the email is invalid or already in use.

We (4 Science) are available to take in charge this improvement by adding a validator around the eperson creation form that highlights the email input as invalid and preventing form submit action. To add this feature and its test we estimate 4 to 8 hours of effort.

**Comment by tdonohue** [comment]
@corrad82-4s : I agree this behavior is not great.  The estimate sounds reasonable, so I'll assign this back to you for you/your team to work on.
